lib/talon/page_controller.ex

```diff
@@ -5,18 +5,30 @@ defmodule Talon.PageController do
       talon = opts[:concern] || raise("concern option required")
       plug :set_concern, talon: talon
 
-      @spec page(Plug.Conn.t, Map.t) :: Plug.Conn.t
-      def page(conn, params) do
-        page = Map.get(params, "page", "dashboard")
-        render(conn, "#{page}.html")
+
+      @spec index(Plug.Conn.t, Map.t) :: Plug.Conn.t
+      def index(conn, params) do
+        page = default_page(conn)
+        render(conn, "#{page}.html") # TODO: use index template (DJS)
+      end
+
+      @spec show(Plug.Conn.t, Map.t) :: Plug.Conn.t
+      def show(conn, params) do
+        page = Map.get(params, "page", default_page(conn))
+        render(conn, "#{page}.html") # TODO: use show template
+      end
+
+      @spec default_page(Plug.Conn.t) :: String.t
+      def default_page(conn) do
+        conn.assigns.talon.concern.dashboard_name()
```

Comment:
I removed this altogether. By the time we're here, the `view` plug has committed to a view using the resource determined by the `load_resource` plug. The `load_resource` plug falls back to a default resource, which is the default_page defined in the concern. The user can override this to change the default page.